apis/ethereum/eth_call.md

```diff
@@ -19,33 +16,43 @@ This is one of the most commonly used API calls. It is used to read from the blo
   - `data`: `DATA` - (optional) Hash of the method signature and encoded parameters. For details see [Ethereum Contract ABI](https://docs.soliditylang.org/en/v0.7.0/abi-spec.html)
 - `QUANTITY|TAG` - integer block number, or the string "latest", "earliest" or "pending" (see the [default block parameter](https://eth.wiki/json-rpc/API#the-default-block-parameter)), OR the `blockHash` (in accordance with [EIP-1898](https://eips.ethereum.org/EIPS/eip-1898)) **NOTE: the parameter is an object instead of a string and should be specified as: `{"blockHash": "0x<some-hash>"}.`** Learn more [here](https://eips.ethereum.org/EIPS/eip-1898).
 
-### Returns
+## Returns
 
-`DATA` - the return value of executed contract.
+`DATA` - the return value of executed contract method.
 
-**Example**
-
-Request
+## **Request example**
 
 ```bash
-curl https://rpc.<REGION>.gateway.fm/v4/ethereum/non-archival/mainnet  \
--X POST \
--H "Authorization: Bearer <YOUR_API_KEY>" \
--H "Content-Type: application/json" \
--d '{"jsonrpc":"2.0","method":"eth_call","params":[{"from": "0x4c88153de66e84c6691fa6bf5b5823530300a942","to": "0x3d14de87b5ade1c61a0b7ca29f7632e6e756b8bf","gas": "0x76c0","gasPrice": "0x9184e72a000","value": "0x0","data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675"}, "latest"],"id":1}'
+curl --location 'https://rpc.eth.gateway.fm' \
+--header 'Content-Type: application/json' \
+--data '{
+    "jsonrpc": "2.0",
+    "method": "eth_call",
+    "params": [
+        {
+            "from": "0x55BA4cB41D0C8F62aB4BB207d0bF9fA42f9FE70d",
```

Comment:
Let's replace "from" to 0x00000000219ab540356cBB839Cbe05303d7705Fa value

Reply:
done